### ISSUE TITLE
Triage quest failed to raise First Aid limit

### ISSUE THREAD

**Issue by GPSRT** [dalaranwow/dalaran-wow#5316]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I completed the Horde side Triage quest on Iceaxe this evening and it did not raise my First Aid limit.

**Expected behaviour**: Tell us what should happen instead
Completing this quest should give me 20 Heavy Runecloth Bandages (which it did), and should raise my First Aid to a maximum of 300 (which it did not do).

**Steps to reproduce the problem**:

1. I've cleared my cache and disabled all addons to try and get Doctor Gregory Victor to give me Triage again, but he does not, likely because I have completed the quest.
2. I have waited 20 minutes to ensure the quest timer would fail and give me the option to redo the quest, but it does not, likely because I have completed it already.
3. First Aid is still at 225/225 and restarting wow does not help.

**Include proofs for this behaviour**
http://www.wowhead.com/quest=6624/triage#comments
The quest Triage should raise my first aid from 225 to 300.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Quest ID: 6623 (Horde side)
NPC ID: 12920 (Doctor Gregory Victor <Trauma Surgeon>)

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_022618_174716](https://user-images.githubusercontent.com/13350632/36700689-8dd0a402-1b1e-11e8-942e-a83afbbe1211.jpg)

Screenshot shows:
- I am level 35 (required for the quest)
- proof I had the quest because the Triage Bandage is on my hot bar (4) which I would need to have picked up the quest for in order to move it there
- completed the quest (I have the 20 Heavy Runecloth Bandages in my bag, but due to the restarting wow lost the message that showed I completed it)
- quest no longer appears (can see NPC in the screenshot without a !)
- talking to him doesn't show the option to pick up the quest again
- my first aid is still 225/225

**Comment by Golganeth** [comment]
Invalid report. The patch notes for this quest changed with 3.1.0 and it's not supposed to raise the player's first aid skill.

Closing up
